File: Editorial/171_Excel_Sheet_Column_Number/Go/2_Left_to_Right.go
func titleToNumber(s string) int {
    result := 0
    n := len(s)
    for i := 0; i < n; i++ {
        result = result * 26
        // In Go, subtracting runes is subtracting ASCII values of characters
        result += (int(s[i]) - int('A') + 1)
    }
    return result
}

File: Editorial/171_Excel_Sheet_Column_Number/JavaScript/2_Left_to_Right.js
var titleToNumber = function (s) {
    let result = 0;
    let n = s.length;
    for (let i = 0; i < n; i++) {
        result = result * 26;
        // In JavaScript, subtracting characters is subtracting ASCII values of characters
        result += s.charCodeAt(i) - "A".charCodeAt(0) + 1;
    }
    return result;
};
